agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -17,7 +17,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 group=default
 dir="/tmp"
 sysinfo_opts_default=( block libvirt kernel_config sos topology ara )
-sysinfo_opts=${sysinfo_opts_default[*]}
+sysinfo_opts=$(IFS=,; echo "${sysinfo_opts_default[*]}")
```

Comment:
Is the only difference between these two that one is space-separated and the other is comma separated? Do they otherwise always contain exactly the same elements?

Reply:
yes, the only difference between two is that sysinfo_opts_default is an array with all the options while sysinfo_opts contains space separated values and they always contain exactly the same elements.